site/en/docs/webstore/cws-dashboard-privacy/index.md

```diff
@@ -53,5 +53,22 @@ does not and select "No, I am not using remote code."
        alt="Screenshot of the remote code declarations field of the privacy field set",
        height="170", width="800" %}
 
+## Certify your data use practices
+
+You must disclose how your extension collects and uses user data. These disclosures include:
+
+* The nature of the data that the extensions collects from users
+* Your certification that the extension complies with the policy on limited use
+
+Your disclosures are displayed to Chrome users, and should be consistent with the existing privacy
+policies that you provide to the Chrome Web Store.
+
+
+**Data usage** Use this field to disclose and certify your data collection practices. Use the first
+group of checkboxes to disclose which types of data your extension collects. Use the second group of
+checkboxes to certify that you comply with each of the disclosure statements.
```

Comment:
This formatting strikes me as a bit odd. Specifically, we have inline bold text that appears to be used as a header immediately followed by an independent sentence. I feel like this should at least be separated by a colon.

Reply:
It's effectively a run-in heading as in the existing material above (like https://screenshot.googleplex.com/at3yQiPwjnARSNg for example). It's just not very well formatted because of the minimalist markdown usage.

I can see doing this with em dashes like we did below the vision pillars https://developer.chrome.com/docs/extensions/mv3/intro/platform-vision/#where-were-headed but maybe it would look better to stick an em space in there in this case.

Reply:
Updated with em spaces. It looks better now.